datarepo-clienttests/src/main/resources/servers/shdev.json

```diff
@@ -15,6 +15,6 @@
     "parameters": ["https://raw.githubusercontent.com/broadinstitute/datarepo-helm-definitions/master/dev/sh/datarepo-api.yaml"]
   },
   "testRunnerServiceAccountFile": "jade-k8-sa.json",
-  "skipDeployment": false,
+  "skipDeployment": true,
```

Comment:
Could you comment why this change is necessary? Thanks!

Reply:
It's not - it's just a change on my personal dev. I decided I'd rather the default be to skip deployment.